src/pages/Homepage/MonthlyGoodHomepage.vue

```diff
@@ -254,11 +299,71 @@ export default {
 
 		};
 	},
+	computed: {
+		heroContentGroup() {
+			return this.content?.page?.contentGroups?.homepageHero ?? null;
+		},
+		heroText() {
+			return this.heroContentGroup?.contents?.find(contentItem => contentItem.key === 'mg-homepage-hero-text');
+		},
+		heroImage() {
+			const mediaObject = this.heroContentGroup?.media?.[0];
+			return {
+				title: mediaObject?.title ?? '',
+				url: mediaObject?.file?.url ?? ''
+			};
+		},
+		heroButton() {
+			return {
+				text: this.heroText?.primaryCtaText ?? '',
+				link: this.heroText?.primaryCtaLink ?? '',
+			};
+		},
+		heroHeadline() {
+			return this.heroText?.headline ?? '';
+		},
+		heroBody() {
+			const text = this.heroText?.bodyCopy ?? '';
+			return documentToHtmlString(text).replace(/\n/g, '<br />');
+		},
+		infoContentGroup() {
+			return this.content?.page?.contentGroups?.homepageContentBlock ?? null;
+		},
+		infoHeadline() {
+			const content = this.infoContentGroup?.contents?.find(contentItem => {
+				return contentItem.key === 'mg-homepage-info-text-header';
+			});
+			const text = content?.richText;
+			// contentful wraps all richText fields with a <p> tag,
+			// which makes them difficult to style as headers,
+			// this removes that wrapping tag
+			const options = {
```

Comment:
very interesting. Nice to see the ability to provide options though!

Reply:
We're using this in a couple of places, it should be extracted to contentful utils